QuizApp/Network/Network.swift

```diff
@@ -0,0 +1,21 @@
+import Network
+import SwiftUI
+import UIKit
+
+class Network: ObservableObject {
+
+    let monitor = NWPathMonitor()
+    let queue = DispatchQueue(label: "Monitor")
+
+    @Published var isConnected = true
```

Comment:
For the sake of exercise, turn isConnected into Combine's AnyPublisher using NWPathMonitor().publisher.